SwrveSDK.podspec

```diff
@@ -14,6 +14,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Sdk/**/*.{m,h}'
   s.public_header_files = 'Sdk/**/*.h'
   s.resources = 'Sdk/Conversation/Resources/*.*'
+  s.dependency 'SwrveSDKCommon'
```

Comment:
Ah yeah, which version of Common we want, a la https://github.com/Swrve/swrve-sdk/blob/release-4_3/ios/SwrveLocationSDK/SwrveLocationSDK.podspec#L16?  Absolutely we do.  Just start at 4.2, or will we move everything to 4.3?